scripts/install/config.default.sh

```diff
@@ -15,6 +15,7 @@ BASE_SITE=
 OCC_PREFIX="/occ/v2/"
 
 URL_PARAMETERS="baseSite,language,currency"
+CURRENCY="USD,EUR"
 
```

Comment:
[MEDIUM]

I guess we don't want to change those defaults for everyone using the installation script in our repo? It might cause the standard apps created with this script to be misconfigured (I guess electronics-spa doesn't use IDR by default? But wants to use EUR and other currencies). Shall we revert this change or make it specific to only OPF?

Reply:
Reverted.